dts/bindings/timer/nordic,nrf-grtc.yaml

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+description: Nordic GRTC (Global RTC)
+
+compatible: "nordic,nrf-grtc"
+
+include:
+  - "base.yaml"
+  - "nordic,split-channels.yaml"
+
+properties:
+    reg:
```

Comment:
This produces the following error in the compliance check:
```
 YAMLLint (indentation):wrong indentation: expected 2 but found 4
File:dts/bindings/timer/nordic,nrf-grtc.yaml
Line:16
Column:5
```
Similar story in `nordic,split-channels.yaml`.